fix: disable conan manager by default (#14171)
renovatebot/renovate: lib/manager/conan/readme.md

<!-- prettier-ignore -->
!!! warning

>     The conan package manager is currently disable by default due to slowness in the Conan API.

Reviewer: ```suggestion
    The Conan package manager is disabled by default due to slowness in the Conan API.
```

>     It is recommended to use it for low volume experimental purposes only until [Issue #14170](https://github.com/renovatebot/renovate/issues/14170) is resolved.

Reviewer: ```suggestion
    We recommend you only enable it for low volume experimental purposes until [issue #14170](https://github.com/renovatebot/renovate/issues/14170) is fixed.
```

Reviewer: I cannot apply the suggestion now that you've edited it.
So you should apply it yourself. 😉
